spec/multi_search/result_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'spec_helper'
+
+describe MeiliSearch::Rails::MultiSearchResult do # rubocop:todo RSpec/FilePath
+  let(:raw_results) do
+    [
+      { 'indexUid' => 'books_index',
+        'hits' => [{ 'name' => 'Steve Jobs', 'id' => '3', 'author' => 'Walter Isaacson', 'premium' => nil, 'released' => nil, 'genre' => nil }],
+        'query' => 'Steve', 'processingTimeMs' => 0, 'limit' => 20, 'offset' => 0, 'estimatedTotalHits' => 1 },
+      { 'indexUid' => 'products_index',
+        'hits' => [{ 'id' => '4', 'href' => 'ebay', 'name' => 'palm pixi plus' }],
+        'query' => 'palm', 'processingTimeMs' => 0, 'limit' => 1, 'offset' => 0, 'estimatedTotalHits' => 2 },
+      { 'indexUid' => 'color_index',
+        'hits' => [
+          { 'name' => 'black', 'id' => '5', 'short_name' => 'bla', 'hex' => 0 },
+          { 'name' => 'blue', 'id' => '4', 'short_name' => 'blu', 'hex' => 255 }
+        ],
+        'query' => 'bl', 'processingTimeMs' => 0, 'limit' => 20, 'offset' => 0, 'estimatedTotalHits' => 2 }
+    ]
+  end
+
+  it 'is enumerable' do
+    expect(described_class).to include(Enumerable)
+  end
```

Comment:
Could use the shorthand `it { is_expected.to include(Enumerable) } `

Reply:
Unfortunately the shorthand runs against `subject`, not `described_class`. rspec tries to create a subject by running `described_class.new` and runs into this:
```
MeiliSearch::Rails::MultiSearchResult 
     Failure/Error:
             def initialize(searches, raw_results)
             // snip
     ArgumentError:
       wrong number of arguments (given 0, expected 2)
```

Moreover if I instantiated a `MultiSearchResult` as `subject` then `include?` would be `Enumerable#include?` which would check if `Enumerable` is one of the hits in the search result.